includes/AMP/Sanitization.php

```diff
@@ -393,8 +393,11 @@ protected function get_sanitizers(): array {
 				 * @todo Enable by default and allow filtering once AMP_Style_Sanitizer does not call AMP_Options_Manager
 				 *       which in turn requires AMP__VERSION to be defined.
 				 */
-				'allow_transient_caching' => false,
-				'use_document_element'    => true,
+				'allow_transient_caching'   => false,
+				'use_document_element'      => true,
+				'dynamic_element_selectors' => [
+					'amp-story-captions',
```

Comment:
Only temporarily needed until we update the AMP plugin dependency.

Reply:
When should we expect the AMP plugin to have these changes?

Reply:
Not sure when the next planned release is there. But the good news is that we're not really blocked by this. It just means we can remove some code once there is a new release.

FWIW, I already submitted the change upstream: https://github.com/ampproject/amp-wp/commit/89c827c4e37d65a4115fce06eb678e991a32b513. So just a matter of waiting for the next release.